subscriptions-api-openapi.yaml

```diff
@@ -0,0 +1,206 @@
+openapi: "3.0.2"
+info:
+  title: CloudEvents Subscriptions API
+  version: "0.1 WIP"
+servers:
+  - url: https://server.example.com/v0.1
+paths:
+  /subscriptions:
+    get:
+      description: Retrieve one or multiple subscriptions
+      parameters:
+        - in: "query"
+          name: "id"
```

Comment:
If we care about being RESTful, I'd prefer that the "fetch by id" be its own resource, with the id being part of the URL path, such as `/subscriptions/{id}`

Reply:
When the POST creation returns a 201 with the Location header having ".../subscriptions?id=1" or that is a URL in the list you obtain with an unqualified GET, that is not more or less RESTful than having the same value in the path. RESTful is when the information from a prior request drives the next request. Knowing where to put data into a URL ahead of time is specifically not RESTful.

TED talk aside:

We can add an option where you can also have the id in the path, but due to OpenAPI's path bias, I would need to factor the operation out, so that either option works.